cmd/notary/tuf.go

```diff
@@ -533,7 +533,7 @@ func tokenAuth(trustServerURL string, baseTransport *http.Transport, gun string,
 	}
 
 	ps := passwordStore{anonymous: readOnly}
-	tokenHandler := auth.NewTokenHandler(authTransport, ps, gun, "push", "pull")
```

Comment:
Oh I missed the `readonly` parameter and agreed your opinion.

On the other hand, do you think it's worth to drop the `readonly` and then use the `actions...` (Just find we have two solutions here)